### PR TITLE
Find potential by structure

### PR DESCRIPTION
**Pull request by liamhuber**
I want to be able to see what Lammps potentials are available for a structure without first creating a job, e.g. [here](https://github.com/pyiron/ironflow/pull/62).

To accomplish this I moved the `list_potentials` and `view_potentials` methods of the Lammps job over to be functions in `pyiron_atomistics.lammps.potential` and exposed them in the init to allow `from pyiron_atomistics.lammps import view_potentials, list_potentials`.

We can discuss whether they should be even more available than that, but for now they're accessible to expert users and don't get in the way of anything else.

### CODE REVIEW

File: pyiron_atomistics/lammps/potential.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
 # Distributed under the terms of "New BSD License", see the LICENSE file.
 
-from __future__ import print_function
+from __future__ import print_function, annotations
```
Comment (pmrv) [inline pyiron_atomistics/lammps/potential.py]:
```suggestion
```

Both the `print` function and type annotations should be in all python versions we support.

**Review by pmrv — APPROVED** [review]
`Project` defines a method `inspect_emperical_potentials` which also allows to do this for users in notebooks, but I don't see an issue with reshuffling the methods here internally.  I'm not actually sure how this is implemented there, but I guess it just returns a lammps potential object.

**Comment by liamhuber** [comment]
I think it's actually a data frame with all the values needed to construct such a potential object actually. Anyhow I also didn't see any issues with an internal reshuffle, and in my other project it is very convenient to have the functionality independent of a project instance as well. Thanks for the timely review!

File: pyiron_atomistics/lammps/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from .potential import view_potentials, list_potentials
```
Comment (jan-janssen) [inline pyiron_atomistics/lammps/__init__.py]: Please use absolute imports!

**Pull request merged**